src/com/twitter/intellij/pants/service/project/PantsOpenProjectProvider.java

```diff
@@ -82,38 +82,27 @@ public Project openProject(@NotNull VirtualFile projectFile, @Nullable Project p
   }
 
   public static OpenProjectTask openTask(Project projectToClose, Project project, boolean forceOpenInNewFrame) {
-    boolean useDefaultTemplate = false;
-    boolean isNewProject = true;
-    boolean sendFrameBack = false;
-    boolean showWelcomeScreen = false;
-    ProjectOpenedCallback callback = null;
-    String projectWorkspaceId = null;
-    FrameInfo frame = null;
-    boolean runConfigurators = false;
-    boolean runConversionsBeforeOpen = false;
-    boolean isProjectCreatedWithWizard = false;
-    boolean isRefreshVfsNeeded = false;
     return new OpenProjectTask(
       forceOpenInNewFrame,
       projectToClose,
-      isNewProject,
-      useDefaultTemplate,
+      true, // isNewProject
```

Comment:
I think the previous style is much more readable. It could be still better if all these variables had been final.

It's a shame that Java doesn't support named arguments.

Reply:
@lukaszwawrzyk what do you think?

Reply:
Named values were ok. People often tend to put scala style named arguments as block comments: `/* isNewProject= */true`, but there are people who don't like it. Generally either way is ok, the change here isn't an improvement in my opinion though.

Reply:
The problem with comments is that forgetting to update them is extremely easy. 

Suppose somebody else renames this argument in the constructor's definition, but keeps its type. Suddenly the comment here becomes incorrect, even though the code will compile and probably will work as previously. Moreover, the person making change to the constructor might be totally unaware that this comment exists and needs to be updated manually.